tasks/Preprocessing/Longbow.wdl

```diff
@@ -238,8 +238,8 @@ task Filter {
 
         longbow filter \
             -v INFO \
-            ~{pbi_arg}~{default="" sep=" --pbi " bam_pbi} \
-            ~{model_arg}~{default="" sep=" --model " model} \
+            ~{pbi_arg}~{default="" bam_pbi} \
```

Comment:
Why?

Reply:
Miniwdl doesn't like using `sep` is normally used for Array variables on non-array variables.  Either the variables need to be changed to Array[] or `sep` needs to be removed